docs/reference/release-notes/7.14.asciidoc

```diff
@@ -3,6 +3,22 @@
 
 Also see <<breaking-changes-7.14,Breaking changes in 7.14>>.
 
+[[known-issues-7.14.2]]
+[discrete]
+=== Known issues
```

Comment:
I think we should include in the text that the issue exists since 7.14.0?

Reply:
I did not find this in the text, only in the anchor above. I think we should simply call it out in the text below.